config/routes.rb

```diff
@@ -3,9 +3,6 @@
 Rails.application.routes.draw do
   root 'home#index'
   resources :users, only: %i[new create] do
-    resources :recipes, only: %i[new create index show]
-  end
-  resources :recipes, only: %i[new create index show] do
-    resources :pictures, only: %i[new create]
+    resources :recipes, only: %i[new create edit index show]
```

Comment:
Is there an `edit`?

Reply:
Oops, not yet. Thanks for pointing that out.